scheduler/src/cook/mesos/api.clj

```diff
@@ -1957,7 +1984,7 @@
         {:summary "Change a job's retry count"
          :parameters {:body-params UpdateRetriesRequest}
          :handler (put-retries-handler conn is-authorized-fn task-constraints)
-         :responses {201 {:schema PosInt
+         :responses {201 {:schema NonNegInt
```

Comment:
I had to change this since `display-retries` now returns `0` if no jobs had failed, but the `failed_only` flag was set. I felt like this made more sense than returning an error status in the case that there were no failed jobs.

Reply:
Is it misleading to respond 201 in this case?

Reply:
Now using 202 Accepted for the empty case